README.md

```diff
@@ -63,52 +58,27 @@ on how you can run your own `go-quai` instance.
 By far the most common scenario is people wanting to simply interact with the Quai
 network: create accounts; transfer funds; deploy and interact with contracts. For this
 particular use-case the user doesn't care about years-old historical data, so we can
-fast-sync quickly to the current state of the network. To do so:
+fast-sync quickly to the current state of the network. 
 
+Using the makefile will preload configuration values from the `network.env` file.
 ```shell
-$ go-quai console
+$ make go-quai
+$ make run-all
 ```
 
-This command will:
- * Start `go-quai` in snap sync mode (default, can be changed with the `--syncmode` flag),
-   causing it to download more data in exchange for avoiding processing the entire history
-   of the Quai network, which is very CPU intensive.
- * Start up `go-quai`'s built-in interactive [JavaScript console](https://geth.ethereum.org/docs/interface/javascript-console),
-   (via the trailing `console` subcommand) through which you can interact using [`web3` methods](https://web3js.readthedocs.io/en/)
-   (note: the `web3` version bundled within `quai` is very old, and not up to date with official docs),
-   as well as `go-quai`'s own [management APIs](https://geth.ethereum.org/docs/rpc/server).
-   This tool is optional and if you leave it out you can always attach to an already running
-   `go-quai` instance with `go-quai attach`.
-
 ### Full node on the Garden test network
 Garden test network is based on the Blake3 proof-of-work consensus algorithm. As such,
 it has certain extra overhead and is more susceptible to reorganization attacks due to the
 network's low difficulty/security.
 
-```shell
-$ geth --garden console
-```
-
-*Note: Older Geth configurations store the Garden database in the `testnet` subdirectory.*
+Modify the `network.env` configuration file to reflect:
+`NETWORK=garden`. You may also want to set `ENABLE_ARCHIVE=true`. Then build and run as previously stated.
 
 ### Configuration
 
-As an alternative to passing the numerous flags to the `geth` binary, you can also pass a
-configuration file via:
-
-```shell
-$ geth --config /path/to/your_config.toml
-```
-
-To get an idea how the file should look like you can use the `dumpconfig` subcommand to
-export your existing configuration:
-
-```shell
-$ geth --your-favourite-flags dumpconfig
-```
-
-*Note: This works only with `geth` v1.6.0 and above.*
+Configuration is handled in `network.env.dist` file. You will need to copy or rename the file to `network.env`. The make commands will automatically pull from this file for configuration changes.
 
+<!--
 #### Docker quick start
 
 One of the quickest ways to get Ethereum up and running on your machine is by using
```

Comment:
if we are updating we should make sure the word Ethereum is completely scrubbed.